query-proxy/proxy.py

```
@@ -0,0 +1,130 @@
+from pygments import highlight
+from pygments.lexers import get_lexer_by_name
+from pygments.formatters import Terminal256Formatter
+import time
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+
+
+class Connection(riffq.BaseConnection):
+    def handle_auth(self, user, password, host, database=None, callback=callable):
```

Ruff (0.13.1)

Unused method argument: `user`

(ARG002)